src/database/dbConnection.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require('pg');
+const url = require('url');
+
+let dbUrl = process.env.DB_LOCAL_URL;
```

Comment:
```suggestion
let dbUrl = process.env.DB_LOCAL_URL;
```
if the process.env.DB_LOCAL_URL equal ''undefined" what the happen?
and if I do not have database ?